sources/api/migration/migrations/v1.18.1/corndog-config-file-v0-1-0/src/main.rs

```diff
@@ -0,0 +1,30 @@
+use migration_helpers::common_migrations::{
+    AddPrefixesMigration, ListReplacement, ReplaceListsMigration,
+};
+use migration_helpers::{migrate, Result};
+use std::process;
+
+fn run() -> Result<()> {
+    migrate(AddPrefixesMigration(vec![
+        "configuration-files.corndog-toml",
+    ]))?;
```

Comment:
Are the values of this config file appropriately added on upgrade? My understanding of `AddPrefixesMigration` is that we may not get the `path` and `template_path` fields.

You can test for this by performing an upgrade and doing something like:

```
apiclient raw -u '/configuration-files'
```

Reply:
From my testing we do get everything inside added, but i'll triple check here in a bit

Reply:
Can you do some additional testing?
- Use apiclient to change the relevant settings and confirm that the config file is re-rended with the changed settings.
- Confirm that the datastore is in the correct state after downgrade and the config file has been removed after downgrade.

Reply:
Sean's comment concerns me. Can you do some additional testing?
- Use apiclient to change the relevant settings and confirm that the config file is re-rended with the changed settings.
- Confirm that the datastore is in the correct state after downgrade and the config file has been removed after downgrade.